kubetest/main.go

```diff
@@ -116,6 +121,7 @@ func defineFlags() *options {
 	flag.StringVar(&o.kubemarkNodes, "kubemark-nodes", "", "Number of kubemark nodes to start")
 	flag.StringVar(&o.logexporterGCSPath, "logexporter-gcs-path", "", "Path to the GCS artifacts directory to dump logs from nodes. Logexporter gets enabled if this is non-empty")
 	flag.BoolVar(&o.multipleFederations, "multiple-federations", false, "If true, enable running multiple federation control planes in parallel")
+	flag.IntVar(&o.parallelism, "parallelism", 1, fmt.Sprintf("How many test runners are run in parallel (1 == serial run, <1 == default parallelism of %d)", defaultParallelism))
```

Comment:
Can we call this ginkgo-parallelism? It will not, for example, control parallelism for `bazel build` or `go test`

Personally I would like the UX for this to be `--parallelism  # auto-select parellelism` vs `--parallelism=10 # exactly 10` (the buildStrategy's --build flag works this way)